dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -101,39 +104,58 @@ def manual_pass
     redirect_to :manual_pass_form
   end
 
+  # get /admin/permissions
   def permissions_form
+    search_term = params[:search_term]
+    permission = params[:permission]
+    if search_term.present?
+      if search_term =~ /^\d+$/
+        @user = User.find(search_term)
```

Comment:
Given the "User Not Found" warning, I'd expect `User.find_by_id` rather than `User.find` so as to not raise an `ActiveRecord::RecordNotFound` exception if no user is found.